packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -0,0 +1,357 @@
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/menu/package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button'
+import Copy from '@spectrum-icons/workflow/Copy';
+import Cut from '@spectrum-icons/workflow/Cut';
+import {Item, Menu, MenuTrigger, Section} from '@react-spectrum/menu';
+import Paste from '@spectrum-icons/workflow/Paste';
+```
+
+---
+category: Collections
+keywords: [menu trigger, dropdown, action menu]
+---
+
+# Menu
+
+<p>{docs.exports.Menu.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Menu', 'MenuTrigger']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/popovers/'}
+  ]} />
+
+## Example
+
+```tsx example
+<MenuTrigger>
+  <ActionButton>
+    Edit
+  </ActionButton>
+  <Menu onAction={(key) => alert(key)}>
+    <Item key="cut">Cut</Item>
+    <Item key="copy">Copy</Item>
+    <Item key="paste">Paste</Item>
+    <Item key="replace">Replace</Item>
+  </Menu>
+</MenuTrigger>
+```
+
+## Content
+
+Menu follows the [Collection Components](../react-stately/collections.html) API, accepting both static and dynamic collections. Similar to the [Picker](Picker.html), Menu accepts `<Item>` elements as children, each with a `key` prop. Basic usage of the Menu, seen in the example above, shows multiple items populated with a string. Static collections, as in this example, can be used when the full list of items is known ahead of time.
+
+Dynamic collections, as shown below, can be used when the items come from an external data source such as an API call, or update over time. Providing the data in this way allows for Menu to automatically cache the rendering of each item, which dramatically improves performance.
+
+As seen below, an iterable list of items is passed to the Menu using the `items` prop. Each item accepts a `key` prop, which is passed to the event handler to identify the selected item. Alternatively, if the item objects contain an id property, this is then used automatically and a key prop is not required. See the [Events](#events) section for more detail on selection and actions.
+
+```tsx example
+function Example() {
+  let menuItems = [
+    {name: 'Cut'},
+    {name: 'Copy'},
+    {name: 'Paste'},
+    {name: 'Replace'}
+  ];
+  return (
+    <MenuTrigger>
+      <ActionButton>
+        Edit
+      </ActionButton>
+      <Menu items={menuItems}>
+        {item => <Item key={item.name}>{item.name}</Item>}
+      </Menu>
+    </MenuTrigger>
+  );
+}
+```
+### Internationalization
+In order to internationalize an Menu, the content provided to all child items should be localized.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the Menu is flipped.
+
+## Sections
+Menu supports sections in order to group options. Sections can be used by wrapping groups of Items in a `Section` component. Each `Section` takes a `title` and `key` prop.
```

Comment:
Were these examples supposed to have selection? The content of the menu items seems to imply that?